pkg/cmd/roachtest/roachtestutil/clusterupgrade/clusterupgrade.go

```diff
@@ -54,15 +54,15 @@ type Version struct {
 // tested, we print the branch name being tested if the test is
 // running on TeamCity, to make it clearer (instead of "<current>").
 func (v *Version) String() string {
+	suffix := ""
```

Comment:
It was kinda annoying debugging test plans without knowing what the "current" version actually is.